src/OgRoleManagerInterface.php

```diff
@@ -46,6 +46,19 @@ public function getDefaultRoles();
   public function getRequiredDefaultRoles();
 
   /**
+   * Returns all the roles of a provided group.
+   *
+   * @param string $entity_type_id
+   *    The entity type id of the group.
+   * @param string $bundle
+   *    The bundle of the group.
+   *
+   * @return \Drupal\og\OgRoleInterface[]
+   *   An array of roles indexed by their ids.
+   */
+  public function loadRolesByGroup($entity_type_id, $bundle);
```

Comment:
Well, I can rename if it's better and more consistent. As for the group, if you check the commits, I implemented it passing the group but reaching the tests I had to create a dummy entity only to get this functionality which makes it unusable so I changed it to retrieve the machine names of the entity keys needed.